### PR TITLE
doc: releases: notes for v15.2.5 octopus

### PR DESCRIPTION
**Pull request by theanalyst**
Signed-off-by: Abhishek Lekshmanan <abhishek@suse.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: doc/releases/octopus.rst [hunk 1]
```diff
@@ -1,3 +1,204 @@
...
+  directories. See the documentation for more information:
+  https://docs.ceph.com/docs/master/cephfs/multimds/
+
+* Monitors now have a config option ``mon_osd_warn_num_repaired``, 10 by default.
+  If any OSD has repaired more than this many I/O errors in stored data a
+ ``OSD_TOO_MANY_REPAIRS`` health warning is generated.
+
+* Now when noscrub and/or nodeep-scrub flags are set globally or per pool,
+  scheduled scrubs of the type disabled will be aborted. All user initiated
+scrubs are NOT interrupted.
```
Comment (smithfarm) [inline doc/releases/octopus.rst:24]: indent this line?

File: doc/releases/octopus.rst [hunk 2]
```diff
@@ -1,3 +1,204 @@
...
+* doc: cephadm batch backport July (1) (`pr#35898 <https://github.com/ceph/ceph/pull/35898>`_, Varsha Rao, Ricardo Marques, Kiefer Chang, Matthew Oliver, Sebastian Wagner, Daniel-Pivonka, Juan Miguel Olmo Martínez, Jan Fajerski, Patrick Donnelly, Adam King, Patrick Seidensal, Michael Fritch, Joshua Schmid)
+* doc: cephadm batch backport July (2) (`pr#36109 <https://github.com/ceph/ceph/pull/36109>`_, Stephan Müller, Patrick Donnelly, Matthew Oliver, Kiefer Chang, Deepika Upadhyay, Volker Theile, Georgios Kyratsas, Adam King, Juan Miguel Olmo Martínez, Dan Mick)
+* doc: cephadm batch backport July (3) (`pr#36265 <https://github.com/ceph/ceph/pull/36265>`_, Matthew Oliver, Sebastian Wagner, Juan Miguel Olmo Martínez, Volker Theile, Varsha Rao, Zac Dover, Adam King, Dimitri Savineau, Patrick Seidensal, Michael Fritch, Joshua Schmid)
+* doc: cephadm batch backport June (3) (`pr#35568 <https://github.com/ceph/ceph/pull/35568>`_, Sebastian Wagner, Kiefer Chang, Zac Dover)
```
Comment (smithfarm) [inline doc/releases/octopus.rst:100]: It doesn't seem right to prefix the cephadm batch backport PRs with `doc: `? How about this format:

```
* cephadm: batch backport June (3) ...
```

File: doc/releases/octopus.rst [hunk 3]
```diff
@@ -1,3 +1,204 @@
...
+  https://docs.ceph.com/docs/master/cephfs/multimds/
+
+* Monitors now have a config option ``mon_osd_warn_num_repaired``, 10 by default.
+  If any OSD has repaired more than this many I/O errors in stored data a
+ ``OSD_TOO_MANY_REPAIRS`` health warning is generated.
+
+* Now when noscrub and/or nodeep-scrub flags are set globally or per pool,
+  scheduled scrubs of the type disabled will be aborted. All user initiated
+scrubs are NOT interrupted.
+
```
Comment (neha-ojha) [inline doc/releases/octopus.rst:25]: Can we please add a note about https://tracker.ceph.com/issues/47297 which fixes a bug in https://tracker.ceph.com/issues/45401?

**Comment by tchaikov** [comment]
jenkins render docs

File: doc/releases/octopus.rst [hunk 4]
```diff
@@ -1,3 +1,204 @@
...
+---------------
+
+* CephFS: Automatic static subtree partitioning policies may now be configured
+  using the new distributed and random ephemeral pinning extended attributes on
+  directories. See the documentation for more information:
+  https://docs.ceph.com/docs/master/cephfs/multimds/
+
+* Monitors now have a config option ``mon_osd_warn_num_repaired``, 10 by default.
+  If any OSD has repaired more than this many I/O errors in stored data a
+ ``OSD_TOO_MANY_REPAIRS`` health warning is generated.
```
Comment (tchaikov) [inline doc/releases/octopus.rst:20]: wrong indent.
```
/home/jenkins-build/build/workspace/ceph-pr-docs/doc/releases/octopus.rst:20: WARNING: Bullet list ends without a blank line; unexpected unindent.
/home/jenkins-build/build/workspace/ceph-pr-docs/doc/releases/octopus.rst:24: WARNING: Bullet list ends without a blank line; unexpected unindent.
```

File: doc/releases/octopus.rst (hunk 3 above)
Comment (theanalyst) [reply doc/releases/octopus.rst:24]: updated with note

Comment (neha-ojha) [reply doc/releases/octopus.rst:24]: thanks @theanalyst!

**Comment by tchaikov** [comment]
jenkins render docs

**Comment by smithfarm** [comment]
Let's add the fix for the LC-enabled bucket issue [1] (which @jdurgin mentioned on the mailing list) once it gets merged and backported to octopus.

[1] https://github.com/ceph/ceph/pull/36997

**Comment by jdurgin** [comment]
AIUI the lifiecycle issue only affected the octopus branch, not 15.2.4, so it doesn't need a release note. @mattbenjamin can confirm

**Comment by theanalyst** [comment]
> AIUI the lifiecycle issue only affected the octopus branch, not 15.2.4, so it doesn't need a release note. @mattbenjamin can confirm

Yes, discussed in standups, this doesn't need a note entry

**Review by neha-ojha — APPROVED** [review]

**Pull request merged**